articles/container-apps/tutorial-event-driven-jobs.md

```diff
@@ -149,6 +204,8 @@ To deploy the job, you must first build a container image for the job and push i
     | `--scale-rule-auth` | The authentication for the scale rule. |
     | `--secrets` | The secrets to use for the job. |
     | `--registry-server` | The container registry server to use for the job. For an Azure Container Registry, the command automatically configures authentication. |
+    | `--mi-user-assigned` | The resource ID of the user-assigned managed identity to assign to the job. |
+    | `--registry-identity` | The resource ID of a managed identity to authenticate with the registry server instead of username/password. If possible, an 'acrpull' role assignemnt will be created for the identity automatically. |
```

Comment:
```suggestion
    | `--registry-identity` | The resource ID of a managed identity to authenticate with the registry server instead of using a username and password. If possible, an 'acrpull' role assignment is created for the identity automatically. |
```